bootstrapvz/common/tasks/image.py

```diff
@@ -0,0 +1,37 @@
+from bootstrapvz.base import Task
+from bootstrapvz.common import phases
+from bootstrapvz.common.tools import log_check_call
+import os
+
+
+class MoveImage(Task):
+	description = 'Moving volume image'
+	phase = phases.image_registration
+
+	@classmethod
+	def run(cls, info):
+		destination = os.path.join(info.manifest.bootstrapper['workspace'], info.image['filename'])
+		import shutil
+		shutil.move(info.volume.image_path, destination)
+		info.volume.image_path = destination
+		import logging
+		log = logging.getLogger(__name__)
+		log.info('The volume image has been moved to ' + destination)
+
+
+class CreateImageTarball(Task):
```

Comment:
Oh, I forgot to mention: this is also a requirement for Oracle Cloud. That's the reason we need to make it a common task.